tests/Unit/Domain/Creators/Test_Sphere.cpp

```diff
@@ -133,24 +143,41 @@ void test_sphere_construction(
   const size_t num_blocks = blocks.size();
   const auto all_boundary_conditions = sphere.external_boundary_conditions();
 
+  // Check total number of external boundaries
+  const size_t num_shells = radial_partitioning.size() + 1;
+  const size_t num_external_boundaries =
+      alg::accumulate(blocks, 0_st, [](const size_t count, const auto& block) {
+        return count + block.external_boundaries().size();
+      });
+  if (which_wedges == ShellWedges::All) {
+    CHECK(num_external_boundaries == (filled ? 6 : 12));
+  } else if (which_wedges == ShellWedges::FourOnEquator) {
+    CHECK(num_external_boundaries ==
+          ((filled ? 2 : 4) + 4 * (1 + num_shells * 2)));
+  } else if (which_wedges == ShellWedges::OneAlongMinusX) {
+    CHECK(num_external_boundaries == ((filled ? 5 : 1) + 1 + num_shells * 4));
+  }
+
   MAKE_GENERATOR(generator);
 
   // verify if adjacent to inner block
```

Comment:
`block` -> `cube` for consistency, can add a comment saying `test_sphere_construction` is not used on Spheres with inner blocks with `sphericity != 0.0`? (Assuming only Spheres with cubical inner blocks are tested)

Reply:
there's a test case with sphericity != 0 as well